src/metabolic_ninja/tasks.py

```diff
@@ -315,16 +317,19 @@ def manipulation_helper(target):
 
 @celery_app.task()
 def cofactor_swap_optimization(pathway, model):
-    pyield = product_yield(pathway.product, model.carbon_source)
+    model.reactions.get_by_id(model.biomass).lower_bound = 0.1
```

Comment:
The value of 0.1 was specific to my example. We need a relative value here. Maybe something like 5%:

```py
with model:
    model.objective = model.biomass
    growth = model.slim_optimize()
model.reactions.get_by_id(model.biomass).lower_bound = 0.05 * growth
```